api/internal/features/lxd/controller/controller.go

```diff
@@ -0,0 +1,111 @@
+package controller
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"github.com/go-fuego/fuego"
+	"github.com/raghavyuva/nixopus-api/internal/features/lxd/service"
+	shared_types "github.com/raghavyuva/nixopus-api/internal/types"
+)
+
+type Controller struct {
```

Comment:
let's follow the same init.go types controller service folder structure as usual